qtm_ip: 192.168.1.145

simid: 6

uris:
- radio://0/80/2M/E7E7E7E7E1
# - radio://0/80/2M/E7E7E7E7E2
# - radio://0/80/2M/E7E7E7E7E3
# - radio://0/80/2M/E7E7E7E7E9

drone_bodies:
- DroneE1
# - DroneE2
# - DroneE3
# - DroneE9

mass:
# E1
- 34.09
# E2
# - 31.06
# E3
- 33.50
# E9
- 32.06

Ts: 0.1

Tsim: 10

T_coeff:
- 1
- 1
- 1
# - 0

alpha:
- 1.0
- 1.0
- 1.0
# - 1.0

PWM0:
# E1
- 40000
# E2
# - 45000
# E3
- 40000
# E9
- 40000

bias:
- [0.0,0.0]
- [0.0,0.0]
- [0.0,-0.0]
- [-0.0,0.0]